CMakeLists.txt

```diff
@@ -102,20 +83,19 @@ if (WIN32)
   list (APPEND CARLA_COMMON_DEFINITIONS _WIN32_WINNT=0x0601) # --- Windows 10 ---
   list (APPEND CARLA_COMMON_DEFINITIONS _CRT_SECURE_NO_WARNINGS)
   list (APPEND CARLA_COMMON_DEFINITIONS HAVE_SNPRINTF)
+  list (APPEND CARLA_COMMON_DEFINITIONS _USE_MATH_DEFINES)
 endif ()
 
 set (CARLA_EXCEPTION_DEFINITIONS)
-
 if (ENABLE_EXCEPTIONS)
+  # Nothing
```

Comment:
### Clean Code Suggestion

In cmake you can use`NOT` inside the `IF` clausules:
https://cmake.org/cmake/help/latest/command/if.html#condition-syntax

Avoiding empty ifs scopes improves readability and performance.

**Resulting Code**
```cmake
if (NOT ENABLE_EXCEPTIONS)
  list (APPEND CARLA_EXCEPTION_DEFINITIONS ASIO_NO_EXCEPTIONS)
  list (APPEND CARLA_EXCEPTION_DEFINITIONS BOOST_NO_EXCEPTIONS)
  list (APPEND CARLA_EXCEPTION_DEFINITIONS LIBCARLA_NO_EXCEPTIONS)
  list (APPEND CARLA_EXCEPTION_DEFINITIONS PUGIXML_NO_EXCEPTIONS)
endif ()
```